.github/workflows/integration-tests.yaml

```diff
@@ -63,7 +63,12 @@ jobs:
       - name: test
         env:
           PIPER_INTEGRATION_GITHUB_TOKEN: ${{secrets.PIPER_INTEGRATION_GITHUB_TOKEN}}
-        run: go test -tags=integration ./integration/...
+        run: |
+          go test -tags=integration ./integration/...
+
+          pushd integration/testdata/TestNpmIntegration/
+          bash run-tests.sh
+          popd
```

Comment:
Instead of the sh file I think you could also use the testContainers to run the piper command inside a container as shown here: https://github.com/SAP/jenkins-library/blob/master/integration/integration_api_cli_test.go#L46